.npmignore

```diff
@@ -0,0 +1,4 @@
+*
+!dist/**
+!package.json
+!README.md
```

Comment:
only files included in the published version of `@baal-contracts` - dist is defined in the [ts.configs here](https://github.com/HausDAO/Baal/pull/44/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R12-R21)